vitrina/orgs/templates/vitrina/orgs/datasets.html

```diff
@@ -0,0 +1,27 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block parent_title %}{% translate "nav.orgs" %}{% endblock %}
+{% block current_title %}{{ organization.title }}{% endblock %}
+
+{% block content %}
+
+	{% include 'vitrina/orgs/tabs.html' %}
+
+    <table class="table is-fullwidth">
+        <tbody>
+        <tr>
+            <td></td>
+            <th>{% translate "Pavadinimas" %}</th>
+            <th>{% translate "Aprašymas" %}</th>
+        </tr>
+        {% for d in datasets %}
+            <tr>
+                <td>{{ forloop.counter }}.</td>
+                <td><a href="{% url 'dataset-detail' d.slug %}">{{ d.title }}</a></td>
```

Comment:
```suggestion
                <td><a href="{{ d.get_absolute_url }}">{{ d.title }}</a></td>
```

https://docs.djangoproject.com/en/3.2/ref/models/instances/#django.db.models.Model.get_absolute_url